include/fluent-bit/flb_config.h

```diff
@@ -300,6 +304,10 @@ enum conf_type {
 #define FLB_CONF_STR_HC_PERIOD                              "HC_Period"
 #endif /* !FLB_HAVE_HTTP_SERVER */
 
+#ifdef FLB_CHUNK_TRACE
+#define FLB_CONF_STR_ENABLE_CHUNK_TRACE      "Enable_Chunk_Trace"
```

Comment:
we need to rename it for understanding, e.g:`http_chunk_trace`, let's discuss a better name for this feature. cc: @niedbalski